Util.py

```diff
@@ -2183,25 +2183,35 @@ def unicode_to_str(s):
   return s
 
 
-def deepcopy(x):
+def deepcopy(x, stop_types=None):
   """
   Simpler variant of copy.deepcopy().
   Should handle some edge cases as well, like copying module references.
 
   :param T x: an arbitrary object
+  :param list[type] stop_types: objects of these types will not be copied
```

Comment:
Wrong type. Should be `list[type]|None`.
Also wrong or misleading documentation. They will be copied, just not deep but shallow (i.e. just the ref).